src/components/ConfirmModal/__tests__/ConfimModal.test.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { ConfirmModal } from "..";
+
+it("Correct render", () => {
+  const { asFragment } = render(
+    <ConfirmModal
+      confirmCallback={() => {}}
+      modalTitle="Confirm title"
+      modalDescr="Confirm descr"
+    />
+  );
+
+  expect(asFragment()).toMatchSnapshot();
+});
+
+it("Work confim callback", () => {
+  const mockCallback = jest.fn();
+  const mockClose = jest.fn();
+
+  render(
+    <ConfirmModal
+      confirmCallback={mockCallback}
+      closeModal={mockClose}
+      modalTitle="Confirm title"
+      modalDescr="Confirm descr"
+    />
+  );
+
+  userEvent.click(screen.getByText(/Confirm/i, { selector: "button" }));
+
+  expect(mockCallback.mock.calls.length).toBe(1);

```

Comment:
The same for others.
```suggestion
  expect(mockCallback).toBeCalledTimes(1);
```